### PR TITLE
update docs for running natively vs in Docker

### PR DESCRIPTION
**Pull request by matt-bernstein**

### CODE REVIEW

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/20450896909180fbe0972c4b9c0ffdd568146d65 is created.

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/3a2fbd1c1eac672d81644e98d78d5e5a2bea81cb is created.

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/efbfb275a4a19488190d79f883447874592f7654 is created.

File: docker-compose.native.yml
[the remark this answers is not in the document]
Comment (matt-bernstein) [reply docker-compose.native.yml:1]: You can develop both ways, and the corresponding readme sections are titled "run natively" and "run in Docker" - this way, the app and workers are running natively and only the off-the-shelf redis and kafka are in docker. Is there a better name that suits this situation?

**Review by hakan458 — APPROVED** [review]

File: docker-compose.native.yml
[the remark this answers is not in the document]
Comment (matt-bernstein) [reply docker-compose.native.yml:1]: gonna go ahead and merge this for now so as not to block the corresponding LSE change

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/6b87b934b51de6e56410f752c8aafbdf3bb4821c is created.

**Pull request merged**